x/incentives/keeper/gauge_test.go

```diff
@@ -254,9 +260,11 @@ func (suite *KeeperTestSuite) TestPerpetualGaugeOperations() {
 	suite.Require().Equal(sdk.Coins(nil), rewardsEst)
 }
 
+// Tests gauge distribution using the gauges by denom keeper.
 func (suite *KeeperTestSuite) TestGaugesByDenom() {
 	// TODO: This is not a good test. We should refactor it to be table driven,
 	// specifying a list of gauges to define, and then the expected result of gauges by denom
+	// TODO: Make issue for this.
```

Comment:
TODO: Make issue for this.

Reply:
This test is being completely refactored to be table-driven in this PR: https://github.com/osmosis-labs/osmosis/pull/1932